### ISSUE TITLE
export does not take 'groupChar' into account

### ISSUE THREAD

**Issue by jze** [frictionlessdata/frictionless-py#1444]
`frictionless export` does not take the `groupChar` property of a table schema into account. This applies to both CSV and JSON export.

In my example the raw value is `8.699,8`. The field in the table schema is described like this:

```json
{        
   "name": "Ansatz 2021 T€",
   "type": "number",
   "decimalChar": ",",
   "groupChar":"."
}
```

While the `decimalChar` specification is taken into account, the `groupChar` specification is apparently ignored.

```
$ frictionless extract --limit-rows 13 --csv  https://opendata.zitsh.de/frictionless/Titelliste_2021.json   | tail -n 1
1,101,41101,Entschädigung der Abgeordneten sowie Übergangsgelder an ausgeschiedene Abgeordnete,8.504.4,8.500.6,8.699.8
```

Framework version: 5.7.1

**Comment by roll** [comment]
Hi @jze,

Currently, `extract` works like a normalization command so when you provide any dialect or field options it's intended to be used to read the data, not to write. 

It's a known problem and we're working on separating two commands:
- extract (explore the data in the console)
- convert (that will support data conversion including this json -> csv case)

**Comment by roll** [comment]
My initial analysis was wrong - it's anyway a bug, I'll release a fix soon

**Comment by roll** [comment]
FIXED in #1459
